src/GrumPHP/Task/SecurityChecker.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Exception\RuntimeException;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * SecurityChecker task
+ */
+class SecurityChecker extends AbstractExternalTask
+{
+    /**
+     * @return string
+     */
```

Comment:
`{@inheritdoc}` is fine here, and in some other places below which implement `TaskInterface`.